### PR TITLE
Adds riffraff artefact upload plugin

### PR DESCRIPTION
**Pull request by kenoir**
This will enable us to push artefacts from a non-Teamcity CI environment  to RiffRaff.

p.s. Jenkins

### CODE REVIEW

File: project/Build.scala [hunk 1]
```diff
@@ -13,12 +17,22 @@ import AssemblyKeys._
...
+      riffRaffUploadManifestBucket := Option("riffraff-builds")
+    )
+
   val commonSettings =
     Seq(
       scalaVersion := "2.11.6",
       scalaVersion in ThisBuild := "2.11.6",
       organization := "com.gu",
-      fork in run := true,
+      fork in run  := true,
```
Comment (theefer) [inline project/Build.scala]: Thank God for that :+1:

**Comment by theefer** [comment]
Is this the same as what TC does? Does it cause double upload when in TC, or is there some magic that prevents it?

**Comment by kenoir** [comment]
@theefer we don't run the correct recipe in TC at the moment to cause this to try and upload to s3 using the plugin.

Tested and works in Jenkins.

**Comment by theefer** [comment]
:+1:

**Pull request merged**